.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -34,7 +34,7 @@ body:
     id: reproduction
     attributes:
       label: Minimal reproduction of the bug/regression with instructions
-      description: Use the [NgRx StackBlitz example](https://stackblitz.com/edit/ngrx-seed) to create a reproduction.
+      description: Use the [NgRx Global Store StackBlitz example](https://stackblitz.com/edit/ngrx-seed) or [NgRx Signal Store StackBlitz example](https://stackblitz.com/edit/ngrx-signal-store-starter) to create a reproduction.
```

Comment:
There's something wrong with the example (I like the quiz btw!).

In `quiz.state.ts`, the 3rd question (line 65) should have id `3` instead of `2`.

Reply:
Fixed it. Thanks, that's what I call a thorough review 😅